ftw/simplelayout/configure.zcml

```diff
@@ -114,4 +114,16 @@
         handler=".handlers.modify_parent_on_block_edit"
         />
 
+    <subscriber
+        for="ftw.simplelayout.interfaces.ISimplelayoutBlock
+             ftw.trash.interfaces.IObjectTrashedEvent"
```

Comment:
This makes `ftw.trash` required for simplelayout, breaking tests -> https://ci.4teamwork.ch/builds/208737/tasks/343649